numpy/core/src/private/ufunc_override.c

```diff
@@ -134,6 +134,14 @@ PyUFunc_WithOverride(PyObject *args, PyObject *kwds,
                 goto fail;
             }
             if (with_override != NULL) {
+                for (j = 0; j < num_override_args; j++) {
```

Comment:
A comment explaining the purpose of this would be nice

Reply:
Something like `/* Like the python __op__ overloads, a type is not visited more than once */`